cachito/common/packages_data.py

```diff
@@ -145,14 +145,13 @@ def load(self, file_name: Union[str, Path]) -> None:
         :type file_name: str or pathlib.Path
         """
         if not os.path.exists(file_name):
-            log.warning("No data is loaded from non-existing file %s.", file_name)
-            return
+            raise CachitoError(f"File {file_name} does not exist.")
 
         with open(file_name, "r", encoding="utf-8") as f:
             data = json.load(f)
             packages = data.get("packages")
             if packages is None:
-                log.warning("Packages data file does not include key 'packages'.")
+                raise CachitoError(f"Packages data file does not include key 'packages'.")
                 return
```

Comment:
Maybe just `data["packages"]`? Under normal circumstances, if the file is valid JSON then it must have the "packages" key. So IMO, if somehow that is not the case, we can just leave that as an uncaught KeyError.